src/install/snowflake/whatsNext.mdx

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
I'd move this info in a new step in the procedure called "Find and use data" like we do for the other OHIs: https://docs.newrelic.com/install/ibm-mq/